### ISSUE TITLE
Request for Multi-Account ID Support in dynatrace_assume_policy for activegate_monitoring_role Module

### ISSUE THREAD

**Issue by priyasingh014** [dynatrace-oss/cloud-snippets#31]
**Is your feature request related to a problem? Please describe.**
Hi Team,

We are interested in using the activegate_monitoring_role Terraform module that you maintain at [this repository](https://github.com/dynatrace-oss/cloud-snippets/tree/main/aws/role-based-access/terraform-templates/activegate_monitoring_role). However, we are facing difficulties integrating it into our project as intended.

Specifically, we would like to include multiple account IDs within the dynatrace_assume_policy_document resource in Terraform.

**Describe the solution you'd like**
As mentioned earlier, we would like the ability to create or append the dynatrace_assume_policy with all the account IDs that we want to monitor using Dynatrace. The current setup only allows for a single account ID to be specified as the monitored_account_id parameter.

**Describe alternatives you've considered**
One alternative we've considered is not using the provided Terraform module and instead developing our own solution.

**Comment by felipmiguel** [comment]
I understand this correctly, you want to have apply the policy to multiple accounts, so it will be solved by applying the module multiple times or by changing the variable to list and apply the policy for all the items in that list. Correct?

**Comment by priyasingh014** [comment]
Yes, for us the ideal case would be the changing the variable to list and apply the policy for all the items in that list.

**Comment by felipmiguel** [comment]
The change doesn't seem complex, however making changes in that parameter may produce some compatibility issues to other customers. Have you considered forking this repository?

**Comment by priyasingh014** [comment]
Hi, we haven't had any discussions within the team about forking the repository yet. I'll take the initiative to discuss this internally. In the meantime, I have a question regarding the first option: 'applying the policy to multiple accounts.' How would that work exactly in our case? Currently, we have only one Active Gate instance running in an AWS account, which sends data for all the accounts where the dynatrace_monitoring_role exists.

**Comment by priyasingh014-ac** [comment]
Hi Team,

Is there any update on this?

Regards,
Priya Singh
